src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -12,6 +12,18 @@
         <fullName>RD2_Entry_Form_Custom_Fields</fullName>
         <label>Recurring Donations Entry Form Custom Fields</label>
     </fieldSets>
+    <fields>
+        <fullName>ACH_Last_4__c</fullName>
```

Comment:
Unless it is requested, let's remove underscores from the API field name (similarly to what has been done for `CardLast4__c`). 
CC @mpusto , @force2b

Reply:
I have underscore because the same field exists on both [oppPayment](https://github.com/SalesforceFoundation/NPSP/blob/4130365eb678378c438e13aff7945a5a8e51180a/src/objects/npe01__OppPayment__c.object#L96) and [Data Import](https://github.com/SalesforceFoundation/NPSP/blob/4130365eb678378c438e13aff7945a5a8e51180a/src/objects/DataImport__c.object#L1336) and both fields have underscore in place. Should I follow the same naming format or update to no underscore

Reply:
That is a good question! Let's keep the underscores for consistency then :(